tests/src/python/test_qgsrendercontext.py

```diff
@@ -464,6 +464,13 @@ def testMapUnitScaleFactor(self):
         sf = r.convertToMapUnits(1, QgsUnitTypes.RenderPixels, c)
         self.assertAlmostEqual(sf, 2.0, places=5)
 
+    def testCustomRenderingFlags(self):
+        rc = QgsRenderContext()
+        rc.setCustomRenderingFlag('myexport', True)
+        rc.setCustomRenderingFlag('omitgeometries', 'points')
+        self.assertTrue(rc.customRenderingFlags()['myexport'])
+        self.assertEqual(rc.customRenderingFlags()['omitgeometries'], 'points')
+
```

Comment:
Also add a test in testFromMapSettings to ensure that the flags are correctly copied from the map settings